picard/pluginmanager.py

```diff
@@ -445,3 +447,22 @@ def _plugins_json_loaded(self, response, reply, error, callback=None):
     # pylint: disable=no-self-use
     def enabled(self, name):
         return True
+
+    def check_update(self, parent=None):
+        if self.available_plugins is None:
+            self.query_available_plugins(partial(self._display_update, parent))
+        else:
+            self._display_update(parent)
+
+    def _display_update(self, parent):
+        test_dictionary = {p.module_name: p.version for p in self.available_plugins}
```

Comment:
This line should be moved inside the `if` branch below.

Also the name isn't great, what about `available_versions` or something like that?

Reply:
Good catch.  No need to assign the variable unless it is going to be used.